jest.config.js

```diff
@@ -7,5 +7,5 @@ module.exports = {
     },
   },
   coveragePathIgnorePatterns: ['src/__tests__/default-orgchart-props.js', 'src/styles', 'src/extension', 'src/data.js'],
-  collectCoverageFrom: ['src/**'],
+  collectCoverageFrom: ['src/**', '!**/locales/**', '!**/all.json', '!**/*.mjs'],
```

Comment:
You could add these to `coveragePathIgnorePatterns` instead. Not sure why the build faild with lack of coverage. But possibly related to this.

Reply:
I don't think it'll fix the coverage either way, this mostly is so that the coverage check doesn't throw a bunch of errors because it can't parse json-files